asyncio_mqtt/client.py

```diff
@@ -360,6 +361,7 @@ def __init__(  # noqa: C901
         if socket_options is None:
             socket_options = ()
         self._socket_options = tuple(socket_options)
+        self.timeout = timeout
```

Comment:
Replace with:

```python
if timeout is None:
    timeout = 10
self._timeout: float = timeout
```

Reply:
I intentionally made `timeout` public. Did you intentionally change it to protected?

Reply:
I guess that was out of habit. 😄 We can keep it public. 👍 Just make a comment about it like this:

```python
# This property is intentionally public since it's safe to modify at any point during the lifetime
# of this instance. Aka: There are no race conditions.
self.timeout = timeout
```

Otherwise, I'll probably forget that it is "safe" and think it was left public by accident. 😅